content/go/notification_ocurrent_events.go.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Managing Notification Occurrence Events services"
 description: "How to interact with the Notification Ocurrence Events services. The SoftLayer_Notification_Occurrence_Event service represents all events with potential to cause a disruption in service.."
-date: "2021-18-01"
+date: "2022-01-13"
```

Comment:
Maybe the date should not be updated to today, just fixed

Reply:
The date was only fixed